cognite/neat/_graph/transformers/_prune_graph.py

```diff
@@ -238,14 +262,21 @@ def _iterate_query(self) -> str:
 
     def _count_query(self) -> str:
         return """
-                SELECT (COUNT(?object) AS ?objectCount)
+                SELECT (COUNT(?object) AS ?count)
                 WHERE {
                     ?subject ?predicate ?object .
                     FILTER (isIRI(?object) && ?predicate != rdf:type)
                     FILTER NOT EXISTS {?object ?p ?o .}
                     }
                 """
 
+    def operation(self, row: ResultRow) -> RowTransformationOutput:
+        row_output = RowTransformationOutput()
+        row_output.remove_triples.append(cast(Triple, row))
+        row_output.instances_removed_count = 1
```

Comment:
Wouldn't this be a  modification? You are not deleting the instance, just removing a connection pointing to a non-existing instance.